cmake/configs/default.cmake

```diff
@@ -90,6 +90,7 @@ hunter_config(ZLIB VERSION 1.2.8-p3)
 hunter_config(ZMQPP VERSION 4.1.2)
 hunter_config(ZeroMQ VERSION 4.1.4-p2)
 hunter_config(caffe VERSION rc3-p2)
+hunter_config(autobahn-cpp VERSION 0.2.0)
```

Comment:
Sorry I don't understand. Probably my whole PR is bad. I just merged my `autobahn-cpp` into your `pkg.autobahn-cpp`. This didn't contains any autobahn specific code, so they were added.
So what I change?